etc/config-template.toml

```diff
@@ -747,6 +747,12 @@
 # optimize-filters-for-hits = false
 
 [raftdb]
+## TiKV supports 2 type raft engines: `rocks` and `raft-engine`.
+## `rocks` is the traditional one, which is RocksDB exactly.
+## `raft-engine` is a new file-based implementation.
+type = "rocks"
+
+## For rocks. 
```

Comment:
it's better to use subconfig here. For example:
```
[raftdb]
type = "rocks"
[raftdb.rocks]
...
[raftdb.raft-engine]
...
```

Reply:
It's old code. It can be removed.